main/midnight/src/library/inc/mxtypes.h

```diff
@@ -38,6 +38,9 @@ typedef std::function<void(void)> MXVoidCallback;
 #define RETURN_IF_NULL( x )  \
     if ( (x) == nullptr ) return
 
+#define RETURN_VALUE_IF_NULL(x, v ) \
```

Comment:
This was the only change needed to build DDR on windows. Expecting a return value from the one method that compile complained about.